content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json

```diff
@@ -79,9 +79,9 @@
     ],
     "su_banner_body": [
         {
-            "value": "<p>To edit this page, add \"\/user\" to the end of your url, and login to Drupal<\/p>\r\n",
+            "value": "<p>To edit this page, <a href=\"\/saml_login\">log in with your SUNetID<\/a>. After you log in, return to the homepage, and use it as a template to begin building your site.<\/p>\r\n",
```

Comment:
Shouldn't this be `/sso/login`?

Reply:
i don't think it matters much in the end. we (john) can change later if we see a need.

Reply:
Did the `/sso/login` work get merged yet?

Reply:
I would prefer `/sso/login`, but I wasn't sure if that was in yet.

Reply:
https://github.com/SU-SWS/stanford_ssp/releases/tag/8.1.2